src/javascript/app_2/App/Components/Layout/Footer/network_status.jsx

```diff
@@ -0,0 +1,19 @@
+import classNames   from 'classnames';
+import PropTypes    from 'prop-types';
+import React        from 'react';
+import { localize } from '_common/localize';
+import Tooltip      from '../../Elements/tooltip.jsx';
+
+const NetworkStatus = ({ status }) => (
+    <div className='network-status-wrapper'>
+        <Tooltip alignment='top' message={localize('Network status: [_1]', [(status.tooltip || 'Connecting to server')])}>
```

Comment:
this is still not localizing the replacement string, it can change to `localize('Connecting to server')`